### DISCUSSION TITLE
Any chance of porting to Miyoo A30?

### DISCUSSION THREAD

**Discussion by redwizardmatt** [OnionUI/Onion#1680] [General]
I know you guys mentioned once in passing that it's something you'd be willing to consider, but I was wondering if you've made a decision on it or not? I've got ample reason to prefer Onion over Spruce and I'm still holding out hope that you guys will find enough joy and love for the A30 to make the leap, but I honestly do understand if that's not the case. I can't imagine how much work that would be for you guys, and I'm fully in support of you doing what's best for you... But I'd be lying if I said it wouldn't make me happy to hear you guys say you're interested in making it happen. 

Anyway, best regards, and happy gaming.

**Comment by hugorosario** [comment]
Oh boy.

This would be awesome.
